# Taken and adapted from lang/python27 port
# Description: Link scripts in the same way Python3 does
# Submitted by: mva

--- Makefile.pre.in.orig	2023-05-26 13:41:44 UTC
+++ Makefile.pre.in
@@ -1003,6 +1003,12 @@ bininstall:	altbininstall
 	(cd $(DESTDIR)$(LIBPC); $(LN) -s tauthon-$(VERSION).pc tauthon2.pc)
 	-rm -f $(DESTDIR)$(LIBPC)/tauthon.pc
 	(cd $(DESTDIR)$(LIBPC); $(LN) -s tauthon2.pc tauthon.pc)
+	-rm -f $(DESTDIR)$(BINDIR)/idle
+	(cd $(DESTDIR)$(BINDIR); $(LN) -s idle$(VERSION) idle2)
+	-rm -f $(DESTDIR)$(BINDIR)/pydoc
+	(cd $(DESTDIR)$(BINDIR); $(LN) -s pydoc$(VERSION) pydoc2)
+	-rm -f $(DESTDIR)$(BINDIR)/2to3
+	(cd $(DESTDIR)$(BINDIR); $(LN) -s 2to3-$(VERSION) 2to3)
 
 # Install the interpreter with $(VERSION) affixed
 # This goes into $(exec_prefix)
